### DISCUSSION TITLE
部署失败提示空间不足才发现怎么镜像这么大了

### DISCUSSION THREAD

**Discussion by miaotouy** [lobehub/lobehub#2873] [Self Hosting] [answered]
我用的轻量服务器，装个宝塔面板，一键安装了推荐项目，部署了newapi，剩个1.24 GB居然不够部署这个lobechat了，我记得之前部署的时候很轻易就装好了
![image](https://github.com/lobehub/lobe-chat/assets/66715028/0abd28c6-9147-4cd8-abb2-761bf60e11ac)

**Comment by arvinxx** [comment, marked as answer]
https://hub.docker.com/r/lobehub/lobe-chat/tags 没变大，建议自己检查下
